packages/table/src/FzTable.vue

```diff
@@ -0,0 +1,277 @@
+    })) ?? [];
+
+const grid = ref()
+
+const staticClasses = ref(['grid'])
+const classes = computed(() => {
+  let res: string[] = [];
+
+  const colLength = props.actions ? columns.length + 1 : columns.length
+
+  switch (colLength) {
+    case 1:
+      res.push('grid-cols-1') 
+      break;
+    case 2:
+      res.push('grid-cols-2') 
+      break;
+    case 3:
+      res.push('grid-cols-3') 
+      break;
+    case 4:
+      res.push('grid-cols-4') 
+      break;
+    case 5:
+      res.push('grid-cols-5') 
+      break;
+    case 6:
+      res.push('grid-cols-6') 
+      break;
+    case 7:
+      res.push('grid-cols-7') 
+      break;
+    case 8:
+      res.push('grid-cols-8') 
+      break;
+    case 9:
+      res.push('grid-cols-9') 
+      break;
+    case 10:
+      res.push('grid-cols-10') 
+      break;
+    case 11:
+      res.push('grid-cols-11') 
+      break;
+    case 12:
+      res.push('grid-cols-12') 
+      break;
+    default:
+      break;
+  }
+  return res;
+})
+
+const colspan = computed(() => {
+  let res: string[] = [];
+
+  const colLength = props.actions ? columns.length + 1 : columns.length
+
+  switch (colLength) {
+    case 1:
+      res.push('col-span-1') 
+      break;
+    case 2:
+      res.push('col-span-2') 
+      break;
+    case 3:
+      res.push('col-span-3') 
+      break;
+    case 4:
+      res.push('col-span-4') 
+      break;
+    case 5:
+      res.push('col-span-5') 
+      break;
+    case 6:
+      res.push('col-span-6') 
+      break;
+    case 7:
+      res.push('col-span-7') 
+      break;
+    case 8:
+      res.push('col-span-8') 
+      break;
+    case 9:
+      res.push('col-span-9') 
+      break;
+    case 10:
+      res.push('col-span-10') 
+      break;
+    case 11:
+      res.push('col-span-11') 
+      break;
+    case 12:
+      res.push('col-span-12') 
+      break;
+    default:
+      break;
+  }
+  return res;
+})
+
+const headerStaticClasses = ['fz__header__cell','sticky','top-0','z-[2]', 'bg-grey-100', 'px-16', 'h-48', 'font-medium', 'flex', 'justify-start', 'items-center']
+const headerClasses = computed(() => {
+
+})
+
+const bodyStaticClasses = ['fz__body', 'z-[1]', 'px-16', 'h-48', 'bg-core-white', 'flex', 'justify-start', 'items-center', 'min-w-min', 'border-b-1', 'border-grey-100']
+const bodyClasses = computed(() => {
+
+})
+
+const getBodyClasses = (column: {props: FzColumnProps, children: FzColumnSlots}, isHeader?: boolean) => {
+  return {
+    relative: !column.props.sticky,
+    sticky: column.props.sticky,
+    ['left-0 z-[2]']: column.props.sticky === 'left',
+    ['z-[3]']: column.props.sticky && isHeader,
+    ['right-0']: column.props.sticky === 'right'
+  }
+}
+
+const updateClasses = () => {
+  const tableWidth = columns.reduce((acc, el, index) => {
+    const colWidth = grid.value.children[index].getBoundingClientRect().width
+    acc+=colWidth
+    return acc
+  }, 0)
+  grid.value.style.width = `${tableWidth}px`
+}
```

Comment:
why is this needed?

Reply:
I think it would be easier to show you, but basically it's needed for the "sticky left or right" part sadly

Reply:
I'll try, I have the feeling that using bindings this will be called a lot, which could become a problem. I'll try that though